### DISCUSSION TITLE
Account type

### DISCUSSION THREAD

**Discussion by samerbuj** [ChristopherKlay/StadiaEnhanced#100] [Q&A] [answered]
Hello, maybe this is a dumb question but do I need Stadia Pro to make the extension work? If that's the case, I can't get the extension to force 2k resolution, don't know what I'm doing wrong.

**Comment by ChristopherKlay** [comment, marked as answer]
You do need Stadia Pro for 2K/4K support - pretty much everything else will work fine on Base.

To force 2K/4K with Pro, simply;

1. Go to your Stadia settings > "Performance" > Switch to "Up to 4K". This will revert back to 1080p if you re-visit the page, but that's a visual bug only - it will still allow for up to 4K once you selected it.
2. In the dropdown menu at the top right, simply change "Resolution" to 1440p (2K) or 2160p (4K)
3. Start a game

It's worth noting that if any issues come up (i.e. the device can't decode the stream fast enough), it will switch to a lower resolution automatically.
